umpf

```diff
@@ -928,7 +928,10 @@ parse() {
 		test -n "${content}" || bailout "${cmd} line without value"
 		if ${STABLE}; then
 			local name="$(<"${STATE}/name")"
-			local version="$(sed "s;${name}/\([0-9]\{8\}\)-.*;\1${VERSION_SEPARATOR}${VERSION};" <<< "${content}")"
+			if [ "${namespace}" = "format_patch" ]; then
+				VERSION="$(sed "s;${name}/[0-9]\{8\}${VERSION_SEPARATOR}\(.*\);\1;" <<< "${content}")"
+			fi
+			local version="$(sed "s;${name}/\([0-9]\{8\}\)${VERSION_SEPARATOR}.*;\1${VERSION_SEPARATOR}${VERSION};" <<< "${content}")"
```

Comment:
That's the wrong place to do this. Try this:
```
format_patch_version() {
       local name="$(<"${STATE}/name")"
       local version="${content#${name}/}"
       echo "${version}" > "${STATE}/version"
       echo "${content}" > "${STATE}/tag"
       echo "$line" >&${series_out}
[...]
```

Reply:
Thanks, that seems to work.